docs/06_how-to-guides/06_abi/01_how-to-use-generator-attributes.md

```diff
@@ -1,6 +1,7 @@
-## ABI/Code generator attributes
+## How to use ABI/Code generator attributes
 Unlike the old ABI generator tool, the new tool uses C++11 or GNU style attributes to mark ```actions``` and ```tables```.
```

Comment:
The contrarian statement regarding previous implementation is unnecessary, instead just tell it how it is _now_

Reply:
Sorry, I was referring to

> **Unlike the old ABI generator tool,** the new tool uses C++11 or GNU style attributes to mark ```actions``` and ```tables```.

Reply:
yeah that was clear from the start